HLTrigger/special/src/HLTL1NumberFilter.cc

```diff
@@ -37,7 +38,8 @@ HLTL1NumberFilter::HLTL1NumberFilter(const edm::ParameterSet& config) :
   inputToken_( consumes<FEDRawDataCollection>(config.getParameter<edm::InputTag>("rawInput")) ),
   period_( config.getParameter<unsigned int>("period") ),
   fedId_(  config.getParameter<int>("fedId") ),
-  invert_( config.getParameter<bool>("invert") )
+  invert_( config.getParameter<bool>("invert") ),
+  useFED1024L1Count_( config.getParameter<bool>("useFED1024L1Count") ) 
 {
```

Comment:
if the new mode only works with the TCDS FED, could you add a check at construction time that `useFED1024L1Count` is set only if `fedId == 1024` ?